core/src/test/scala/org/apache/spark/shuffle/ShuffleChecksumTestHelper.scala

```diff
@@ -37,4 +39,18 @@ trait ShuffleChecksumTestHelper {
     assert(ShuffleChecksumUtils.compareChecksums(numPartition, algorithm, checksum, data, index),
       "checksum must be consistent at both write and read sides")
   }
+
+  def getRowBasedChecksumValues(rowBasedChecksums: Array[RowBasedChecksum]): Array[Long] = {
+    if (rowBasedChecksums.isEmpty) {
+      Array.empty
+    } else {
+      rowBasedChecksums.map(_.getValue)
+    }
+  }
+
+  def createPartitionRowBasedChecksums(
+      numPartitions: Int,
+      checksumAlgorithm: String): Array[RowBasedChecksum] = {
```

Comment:
Given it's test only, do we really need to respect the checksum algorithm config? Can we just pick a simplest algorithm?

Reply:
Sounds good. As we don't compare and verify different checksumAlgorithms here, set as `ADLER32` which is the simplest in the valid set.